### DISCUSSION TITLE
New here

### DISCUSSION THREAD

**Discussion by Briatico** [community/community#129987] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How can I send a message to Anton Lashkov? Where is the notepad app that was in play store?

**Comment by ebndev** [comment, marked as answer]
Hi @Briatico, thanks for being a part of the GitHub Community!

It looks like you may have ended up here by clicking “Send feedback” on a user’s profile, is that right? That link is meant for users to provide feedback for Achievements, which leads to the Profile category of GitHub Discussions, a public forum.

We are working on a solution to that however, you might want to try going back to the project you were seeking support for and open an issue in the project's repository or contact them on their website / social media. There is currently no way to contact users directly on GitHub.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!
